src/app/aladin-map/aladin-map.component.html

```diff
@@ -1 +1,3 @@
-<div id="aladin-lite-div" style="width:400px;height:400px;"></div>
+<!DOCTYPE html>
```

Comment:
Remove doctype directive from the component?
I think the doctype directive belongs only in `index.html`, not in components.
Otherwise you'll create multiple doctype directives in the rendered HTML page where the components are inserted.